### DISCUSSION TITLE
Labels

### DISCUSSION THREAD

**Discussion by kubraaksux** [bounswe/bounswe2023group1#21] [Show and tell]
We now have 43 labels, i think they seem enough for now and for a while. Tried to consider future occasions too but im open to any suggestions.

**Comment by volcaniqueo** [comment]
Lifecycle/Active & Lifecycle/Closed seem to be unnecessary to me since we have already two separated places for open & closed issues.

**Reply by kubraaksux** [reply]
I saw it is used in good repositories for example Kubernetes. we have wide range of status labels, and i thought lifecycle label can be more summarised. i think it can be useful after the milestones and if not preferred we may not use it till then. what you think about this perspective?
folks lets discuss that

**Reply by volcaniqueo** [reply]
alright, let's discuss this and general label usage principles at our weekly meeting.

**Reply by kubraaksux** [reply]
yeah thats very good idea!

**Comment by alperenDagi** [comment]
As we talked about in the last lesson, effort label should be added.

**Comment by alitpc25** [comment]
How about adding labels such as Component: Backend, Component: Frontend, Component: Mobile?
